### PR TITLE
enableLearningMode start

### PR DESCRIPTION
**Pull request by btopro**

### CODE REVIEW

**Comment by Griffin-Sullivan** [comment]
@btopro it's working as far as I've tested. Still want to add changes to settings that will allow me to disable transcript completely.

File: elements/a11y-media-player/a11y-media-player.js [hunk 1]
```diff
@@ -2883,6 +2896,11 @@ class A11yMediaPlayer extends FullscreenBehaviors(SimpleColors) {
     }
   }
 
+  enableLearningMode() {
```
Comment (btopro) [inline elements/a11y-media-player/a11y-media-player.js:2899]: remove this function unless it serves a purpose. then looks good!

**Review by btopro — COMMENTED** [review]
Update the line in question and I'll accept this

**Comment by Griffin-Sullivan** [comment]
> Update the line in question and I'll accept this

Ok done

**Pull request merged**